dlrover/python/common/constants.py

```diff
@@ -71,6 +71,12 @@ class ElasticJobLabel(object):
     RELAUNCH_COUNT = "elasticjob.dlrover/relaunch-count"
 
 
+class SchedulingLabel(object):
+    RACK_GROUP = "scheduling/rack-group"  # int
```

Comment:
As we discussed earlier, strongly infrastructure-related definitions should not appear in DLRover. Avoid using terms like 'rack'. 
RACK_GROUP -> RANK_GROUP
RACK_GROUP_SIZE -> RANK_GROUP_SIZE
RACK_ID -> RANK_GROUP_ID